spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskControllerTests.java

```diff
@@ -151,6 +157,59 @@ public void testSave() throws Exception {
 	}
 
 	@Test
+	public void testCompose() throws Exception {
+		assertEquals(0, repository.count());
+		TaskDefinition testTask = new TaskDefinition("task", "task");
+		repository.save(testTask);
+		appRegistry.save("task", ApplicationType.task, new URI("http://fake.example.com/"),null);
+		mockMvc.perform(
+				post("/tasks/definitions/compose").param("name", "myTask").param("definition", "task")
+						.accept(MediaType.APPLICATION_JSON)).andDo(print())
```

Comment:
Please don't `print()`, we have enough of those already (which should be removed IMHO)